subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -734,27 +779,56 @@ void ull_conn_iso_start(struct ll_conn *acl, uint32_t ticks_at_expire, uint16_t
 				       cig->lll.window_widening_periodic_us_frac;
 		ticks_periodic  = EVENT_US_FRAC_TO_TICKS(iso_interval_us_frac);
 		ticks_remainder = EVENT_US_FRAC_TO_REMAINDER(iso_interval_us_frac);
+#endif /* CONFIG_BT_CTLR_PERIPHERAL_ISO */
 
-		/* Adjust CIG offset and reference point ahead one interval */
-		cig_offset_us -= EVENT_TICKER_RES_MARGIN_US;
-		cig_offset_us -= EVENT_JITTER_US;
-		cig_offset_us -= ready_delay_us;
-		cig_offset_us += (acl->lll.interval * CONN_INT_UNIT_US);
-
-		cig->cig_ref_point += (acl->lll.interval * CONN_INT_UNIT_US);
```

Comment:
It seems like you removed the "ahead one interval". This was made because not all implementations will be able to start the CIG in due time on the ACL instant. We need to put this under configuration.

Reply:
Not sure, will check. OK to remove for now then.